packages/cli/lib/supportsColor.js

```diff
@@ -0,0 +1,122 @@
+const process = require('process');
+const os = require('os');
+const tty = require('tty');
+const hasFlag = require('./hasFlag');
+
+const { env } = process;
+
+function translateLevel(level) {
+  if (level === 0) {
+    return false;
```

Comment:
I just noticed, but this returns either `false` or an object. Maybe we should update this function to consistently return an object?

Reply:
I would hesitate to change this, since the function `createSupportsColor` returns `translateLevel`, and `createSupportsColor` is critical to the util. If we were to change it, I think it would require a fair bit of testing, and I'm not sure the juice is worth the squeeze.

Reply:
Just discussed over Slack, will change now.